src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -404,10 +405,12 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
         md->flags |= UCT_IB_MLX5_MD_FLAG_KSM;
     }
 
+#if HAVE_DEVX_ODP
```

Comment:
can we disable this in runtime bu DEVX flag
because user can compile with DEVX but turn it off

Reply:
Yes, we do. It is DEVX flow

Reply:
so if HAVE_DEVX_ODP is like disabling it for now?
maybe add some comment here?

Reply:
Yeah.
BTW shouldn't we open RM - to close the gap?
Then could add /* TODO pending resolution of issue NN ...

Reply:
Yep, can you pls open?